lib/rubocop/ast_node.rb

```diff
@@ -0,0 +1,17 @@
+# encoding: utf-8
+
+require 'astrolabe/node'
+
+module Astrolabe
+  # RuboCop's extensions to Astrolabe::Node (which extends Parser::AST::Node)
+  #
+  # Contribute as much of this as possible to the `astrolabe` gem
+  # If any of it is accepted, it can be deleted from here
+  #
+  class Node
+    def multiline?
+      expr = loc.expression
+      expr && expr.source.include?("\n")
```

Comment:
The issue with strings sounds very doubtful. If you have a string like `"a line\nanother line"`, then the source should include `'\n'` but not `"\n"`, if you know what I mean. But the suggested code looks like it should be much more efficient on large nodes. I'm updating it now... thanks.